src/ledger/cleanup_service.zig

```diff
@@ -11,7 +11,7 @@ const Slot = sig.core.Slot;
 
 const BlockstoreDB = ledger.BlockstoreDB;
 const BlockstoreReader = ledger.reader.BlockstoreReader;
-const BlockstoreWriter = ledger.writer.BlockstoreWriter;
+const LedgerResultWriter = ledger.result_writer.LedgerResultWriter;
```

Comment:
Should `BlockstoreReader` be also updated to `LedgerResultReader`? To keep it symmetric.